src/python_testing/matter_testing_infrastructure/chip/testing/pics.py

```
@@ -64,11 +64,22 @@ def parse_pics(lines: typing.List[str]) -> dict[str, bool]:
 
 
 def parse_pics_xml(contents: str) -> dict[str, bool]:
-    pics = {}
+    pics: dict[str, bool] = {}
     mytree = ET.fromstring(contents)
     for pi in mytree.iter('picsItem'):
-        name = pi.find('itemNumber').text
-        support = pi.find('support').text
+        name_elem = pi.find('itemNumber')
+        support_elem = pi.find('support')
+
+        # Skip if either element is None
+        if name_elem is None or support_elem is None:
```

This is changing the behaviour - before this would throw an error if either of these didn't exist (granted, a fairly hard to understand error), but now they get silently ignored. Can you throw an error here? The PICS XML format is strict - these should be present.

---

Sure, I've enhanced error message here.